README.md

```diff
@@ -222,6 +226,14 @@ Install the pre-commit hooks with `pip install pre-commit && pre-commit install`
 ## Acknowledgments
 The Habitat project would not have been possible without the support and contributions of many individuals. We would like to thank Xinlei Chen, Georgia Gkioxari, Daniel Gordon, Leonidas Guibas, Saurabh Gupta, Or Litany, Marcus Rohrbach, Amanpreet Singh, Devendra Singh Chaplot, Yuandong Tian, and Yuxin Wu for many helpful conversations and guidance on the design and development of the Habitat platform.
 
+
+## External Contributions
+
+* PyRobot Noise Model -- [PyRobot](https://github.com/facebookresearch/pyrobot#citation)
+
+    Contributes the noise model used for the noisy control functions named `pyrobot_*` and defined in `habitat_sim/agent/pyrobot_noisy_controls.py`
+
```

Comment:
Can we restructure this bit? 

"If you use the PyRobot noise models, please cite: 
blah

Noisy control function named `pyrobot_*` defined in `blah`.

Reply:
Done